prowler/providers/aws/services/awslambda/awslambda_function_not_directly_publicly_accessible_via_elbv2/awslambda_function_not_directly_publicly_accessible_via_elbv2.py

```diff
@@ -0,0 +1,29 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.awslambda.awslambda_client import awslambda_client
+from prowler.providers.aws.services.elbv2.elbv2_client import elbv2_client
+
+
+class awslambda_function_not_directly_publicly_accessible_via_elbv2(Check):
+    def execute(self):
+        findings = []
+
+        public_lambda_functions = {}
+        for target_group in elbv2_client.target_groups:
+            if target_group.public and target_group.target_type == "lambda":
```

Comment:
Should add a verification to ensure that the function has a policy that makes really public. Please see `awslambda_function_not_publicly_accessible` check to add the same verification before adding to `public_lambda_function`

Reply:
done